app/views/templates/_template.html.slim

```diff
@@ -0,0 +1,11 @@
+tr
+  td class="question"
```

Comment:
code which prevents unauthenticated user to access templates:
in controller:
before_action :authenticate_user!

in routes:
devise_scope :user do
    authenticated :user do
      root 'users#show', as: :authenticated_root
    end
  
    unauthenticated do
      root 'devise/sessions#new', as: :unauthenticated_root
    end
  end
I tried to go to the template as unauthenticated and received: get '*a' => 'static_pages#error'